config/firewalld.service.in

```diff
@@ -8,11 +8,8 @@ Documentation=man:firewalld(1)
 
 [Service]
 EnvironmentFile=-/etc/sysconfig/firewalld
-ExecStart=@sbindir@/firewalld --nofork --nopid $FIREWALLD_ARGS
+ExecStart=@sbindir@/firewalld --nofork --nopid --log-channel=syslog $FIREWALLD_ARGS
```

Comment:
Users have come to expect logs in `/var/log/firewalld`. Will it cause friction to change that? Ideally distributions would do whatever makes sense for them.

Reply:
Added a build-time parameter to build firewalld with syslog only as the default log target. Without enabling it, everything remains backward compatible.